app/view_models/auction_list_item.rb

```diff
@@ -0,0 +1,193 @@
+class AuctionListItem
```

Comment:
super long, but all the methods we need for `auctions#index` are here! (now more presenter or `AuctionViewModel` being used 💃 )